pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -246,21 +246,6 @@ def restore_loops(self) -> None:
                 f" but you have set Trainer(max_epochs={self.trainer.max_epochs})."
             )
 
-        # Division deals with global step stepping once per accumulated batch
-        # Inequality deals with different global step for odd vs even num_training_batches
-        self.trainer.accumulate_grad_batches = self.trainer.accumulation_scheduler.get_accumulate_grad_batches(
-            self.trainer.current_epoch
-        )
-        n_accum = 1 if self.trainer.accumulate_grad_batches is None else self.trainer.accumulate_grad_batches
```

Comment:
Is this not necessary anymore? Does `accumulate_grad_batches` return always 1?

Reply:
GradientScheduler returns 1 by default